vector/src/main/res/values/strings.xml

```diff
@@ -2374,7 +2374,10 @@ Not all features in Riot are implemented in RiotX yet. Main missing (and coming
     <string name="invite_users_to_room_title">Invite Users</string>
     <string name="invitation_sent_to_one_user">Invitation sent to %1$s</string>
     <string name="invitations_sent_to_two_users">Invitations sent to %1$s and %2$s</string>
-    <string name="invitations_sent_to_three_and_more_users">Invitations sent to %1$s and %2$d more</string>
+    <plurals name="invitations_sent_to_one_and_more_users">
+        <item quantity="two">Invitations sent to %1$s and 2 more</item>
+        <item quantity="other">Invitations sent to %1$s and %2$d more</item>
+    </plurals>
```

Comment:
This is not correct.
You have to write this:

```
    <plurals name="invitations_sent_to_one_and_more_users">
        <item quantity="one">Invitations sent to %1$s and one more</item>
        <item quantity="other">Invitations sent to %1$s and %2$d more</item>
    </plurals>
```

Even if in your code you will never have the case with quantity == 1